client/pom.xml

```diff
@@ -478,6 +478,12 @@
                 <copy todir="${basedir}/target/generated-webapp/WEB-INF/classes/scripts">
                   <fileset dir="${basedir}/../scripts"/>
                 </copy>
+                <copy todir="${basedir}/target/generated-webapp/WEB-INF/classes/systemvm/js">
```

Comment:
My understanding about unit testing is similar to mentioned in this blog http://simpleprogrammer.com/2010/10/15/the-purpose-of-unit-testing/